README.md

```diff
@@ -128,6 +134,8 @@ These options are used to override the focus trap's default behavior for this pa
 
 - **returnFocus** `{boolean}`: Default: whatever you chose for `createOptions.returnFocusOnDeactivate`.
 - **onDeactivate** `{() => void | null | false}`: Default: whatever you chose for `createOptions.onDeactivate`. `null` or `false` are the equivalent of a `noop`.
+- **onPostDeactivate** `{(value: unknown) => void | null | false}`: Default: whatever you chose for `createOptions.onPostDeactivate`. `null` or `false` are the equivalent of a `noop`.
+- **checkCanFocusTrigger** `{(trigger: HTMLElement | SVGElement) => Promise<unknown> | null | false}`: Default: whatever you chose for `createOptions.checkCanFocusTrigger`.
```

Comment:
Same here for the `value: unknown`, `void | null | false`, and "`null` or `false` are the equivalent of a `noop`." at the end as in my previous comments.